core/src/scheme/scheme.rs

```diff
@@ -133,6 +134,21 @@ impl Scheme {
         Ok((db, root))
     }
 
+    fn initialize_metadata<DB: AsHashDB>(&self, mut db: DB, mut root: H256) -> StateResult<(DB, H256)> {
+        {
+            let global_metadata = Metadata::new();
+
+            let mut t = TrieFactory::from_existing(db.as_hashdb_mut(), &mut root)?;
+            let address = MetadataAddress::new();
+
+            let r = t.insert(&*address, &global_metadata.rlp_bytes());
+            debug_assert_eq!(Ok(None), r);
+            r?;
+        }
+
+        Ok((db, root))
+    }
+
```

Comment:
I suggest moving this change to a separated commit.
I think it's too important to be aggregated with removing tracket and vm.